NotebookExamples/csharp/Samples/Repo Statistics.ipynb

```diff
@@ -254,17 +254,31 @@
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
+     "ename": "Unhandled Exception",
+     "evalue": "Sequence contains more than one element",
```

Comment:
Why the error?

Reply:
I accidentally checked that in

Reply:
Interestingly this reproduces for me now. There might be a bug in the notebook

Reply:
Which version are you using?